lib/msal-core/package.json

```diff
@@ -29,6 +29,7 @@
     "clean": "shx rm -rf dist docs lib-commonjs lib-es6",
     "doc": "typedoc --mode modules --excludePrivate --excludeProtected --out ./docs ./src/ --gitRevision dev",
     "build:modules": "tsc && tsc -m es6 --outDir lib-es6",
+    "build:modules:watch": "tsc -m es6 --outDir lib-es6 --watch",
```

Comment:
just so I understand, as you make changes this will update the build files?

Reply:
Yep.